### PR TITLE
Feature/e UI 3378 change link display phillip whitaker

### PR DESCRIPTION
**Pull request by phillip-whitaker**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-3378

### Change description ###

Provide a change link if the user has 'pui-financial-manager' role.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by Mo-Lala84 — DISMISSED** [review]
nearly made me raise a comment, but all good ;-)

File: src/styles.scss [hunk 1]
```diff
@@ -23,6 +23,10 @@ td.govuk-table__cell {
   white-space: pre-wrap;
 }
 
+td.govuk-table__cell a.govuk-link {
+  float: right;
```
Comment (paulhowes-HMCTS) [inline src/styles.scss:27]: Pity there's not a pull-right or rfloat class to defined in styles.scss

File: src/organisation/containers/organisation/organisation.component.spec.ts [hunk 2]
```diff
@@ -174,4 +189,61 @@ describe('OrganisationComponent', () => {
...
+    it('should show the \'Change\' link when false', () => {
+      component.showChangePbaNumberLink = true;
+      fixture.detectChanges();
+
+      const changeLink = fixture.debugElement.query(CHANGE_LINK);
+
+      expect(changeLink).toBeTruthy();
+    });
+
+    it('should not show the \'Change\' link when false', () => {
```
Comment (paulhowes-HMCTS) [inline src/organisation/containers/organisation/organisation.component.spec.ts:213]: backticks would look better here to remove the need to put \'

File: src/organisation/containers/organisation/organisation.component.spec.ts [hunk 3]
```diff
@@ -174,4 +189,61 @@ describe('OrganisationComponent', () => {
...
+
+    it('should set to true when the user has pui-finance-manager and there are PBA accounts', () => {
+      userIsPuiFinanceManager = true;
+      component.organisationPaymentAccount = [
+        'PBA00000'
+      ];
+
+      component.canShowChangePbaNumbersLink();
+
+      fixture.detectChanges();
```
Comment (paulhowes-HMCTS) [inline src/organisation/containers/organisation/organisation.component.spec.ts:233]: Does this need change detection? What would happen if this line wasn't here?

**Review by paulhowes-HMCTS — CHANGES_REQUESTED** [review]
Few initial thoughts.

File: src/organisation/containers/organisation/organisation.component.spec.ts (hunk 3 above)
Comment (phillip-whitaker) [reply src/organisation/containers/organisation/organisation.component.spec.ts]: done

File: src/organisation/containers/organisation/organisation.component.spec.ts (hunk 2 above)
Comment (phillip-whitaker) [reply src/organisation/containers/organisation/organisation.component.spec.ts]: changed

File: src/organisation/containers/organisation/organisation.component.spec.ts (hunk 3 above)
Comment (phillip-whitaker) [reply src/organisation/containers/organisation/organisation.component.spec.ts]: You were right, it didn't need this so I removed.

File: src/styles.scss (hunk 1 above)
Comment (phillip-whitaker) [reply src/styles.scss:27]: There wasn't anything there already to use so I added this.

Comment (paulhowes-HMCTS) [reply src/styles.scss:27]: If we need any more float rights we should consider adding a generic style in future.

**Review by paulhowes-HMCTS — CHANGES_REQUESTED** [review]
I will be happy to approve once the Jenkins build process has passed and a preview Url is generated.

**Review by Mo-Lala84 — DISMISSED** [review]
lgtm

**Review by paulhowes-HMCTS — APPROVED** [review]
All good. Approved.

**Comment by phillip-whitaker** [comment]
Closing as these changes have been placed in: https://github.com/hmcts/rpx-xui-manage-organisations/pull/655

**Pull request closed without merging**